utilities/pipelines/staticValidation/psrule/.ps-rule/storage-firewall-suppress.Rule.yaml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
This will trigger all workflows. I'd suggest a separate PR for this and also to agree on how to suppress ad hoc rules as per conversation started by PR

Reply:
Update: this has triggered all workflows 😄

Reply:
> This will trigger all workflows. I'd suggest a separate PR for this and also to agree on how to suppress ad hoc rules as per conversation started by draft PR #3810 

Fair point. Something we can still do. I hope the item is somewhere on the agenda. My main concern was to get the module unblocked as I was unable to publish any changes since we enforced the security rules. If we change it subsequently to the approach in the linked draft PR it's more than welcome 💪